src/Mod/Path/PathScripts/PathDressupLeadInOut.py

```diff
@@ -33,46 +33,127 @@
-        obj.addProperty("App::PropertyBool", "UseMachineCRC", "Path", QtCore.QT_TRANSLATE_NOOP("App::Property", "Use Machine Cutter Radius Compensation /Tool Path Offset G41/G42"))
-        obj.addProperty("App::PropertyDistance", "Length", "Path", QtCore.QT_TRANSLATE_NOOP("App::Property", "Length or Radius of the approach"))
-        obj.addProperty("App::PropertyEnumeration", "StyleOn", "Path", QtCore.QT_TRANSLATE_NOOP("Path_DressupLeadInOut", "The Style of LeadIn the Path"))
-        obj.StyleOn = ["Arc", "Tangent", "Perpendicular"]
-        obj.addProperty("App::PropertyEnumeration", "StyleOff", "Path", QtCore.QT_TRANSLATE_NOOP("Path_DressupLeadInOut", "The Style of LeadOut the Path"))
-        obj.StyleOff = ["Arc", "Tangent", "Perpendicular"]
-        obj.addProperty("App::PropertyEnumeration", "RadiusCenter", "Path", QtCore.QT_TRANSLATE_NOOP("Path_DressupLeadInOut", "The Mode of Point Radiusoffset or Center"))
-        obj.RadiusCenter = ["Radius", "Center"]
+        obj.addProperty(
+            "App::PropertyLink",
+            "Base",
+            "Path",
+            QT_TRANSLATE_NOOP("App::Property", "The base path to modify"),
+        )
+        obj.addProperty(
+            "App::PropertyBool",
+            "LeadIn",
+            "Path",
+            QT_TRANSLATE_NOOP("App::Property", "Calculate roll-on to path"),
+        )
+        obj.addProperty(
+            "App::PropertyBool",
+            "LeadOut",
+            "Path",
+            QT_TRANSLATE_NOOP("App::Property", "Calculate roll-off from path"),
+        )
+        obj.addProperty(
+            "App::PropertyBool",
+            "KeepToolDown",
+            "Path",
+            QT_TRANSLATE_NOOP("App::Property", "Keep the Tool Down in Path"),
+        )
+        obj.addProperty(
+            "App::PropertyBool",
+            "UseMachineCRC",
+            "Path",
+            QT_TRANSLATE_NOOP(
+                "App::Property",
+                "Use Machine Cutter Radius Compensation /Tool Path Offset G41/G42",
+            ),
+        )
+        obj.addProperty(
+            "App::PropertyDistance",
+            "Length",
+            "Path",
+            QT_TRANSLATE_NOOP("App::Property", "Length or Radius of the approach"),
+        )
+        obj.addProperty(
+            "App::PropertyEnumeration",
+            "StyleOn",
+            "Path",
+            QT_TRANSLATE_NOOP("App::Property", "The Style of motion into the Path"),
+        )
+        obj.StyleOn = lead_styles
+        obj.addProperty(
+            "App::PropertyEnumeration",
+            "StyleOff",
+            "Path",
+            QT_TRANSLATE_NOOP("App::Property", "The Style of motion out of the Path"),
+        )
+        obj.StyleOff = lead_styles
+        obj.addProperty(
+            "App::PropertyEnumeration",
+            "RadiusCenter",
+            "Path",
+            QT_TRANSLATE_NOOP(
+                "App::Property", "The Mode of Point Radiusoffset or Center"
```

Comment:
Is this a typo?

Reply:
I just copied this verbatim. Fixing bugs in the op itself is out of scope for this PR.